modules/Bio/EnsEMBL/Variation/Phenotype.pm

```diff
@@ -82,12 +82,13 @@ sub new {
   my $caller = shift;
   my $class  = ref($caller) || $caller;
   my $self = $class->SUPER::new(@_);
-  my ($dbID, $description, $name, $class_attrib) = rearrange([qw(dbID DESCRIPTION NAME CLASS_ATTRIB)], @_);
+  my ($dbID, $stable_id, $description, $name, $class_attrib) = rearrange([qw(dbID stable_id DESCRIPTION NAME CLASS_ATTRIB)], @_);
```

Comment:
I know this is mega-neurotic, but it bothers me more than it should that stable_id isn't in caps here